Calculation.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CalculatorLibrary
+{
+    public class Calculation
+    {
+        protected double num1 = 0;
+        protected double num2 = 0;
+        public double Result { get; protected set; } = 0;
+        protected string op = "";
+
+        public Calculation(double num1, double num2, double result, string op)
+        {
+            this.num1 = num1;
+            this.num2 = num2;
+            this.Result = result;
```

Comment:
![image](https://user-images.githubusercontent.com/108711476/202833566-8bac759a-68dc-4b7e-911d-1c4427f4c2fd.png)

"this" isn't necessary in line 20